cmd/generate.go

```diff
@@ -227,6 +227,12 @@ func updateDiff(config *internal.Config, newMigrationFilePath string, initial bo
 	}
 
 	if initial {
+		// Verify the schema is correct by applying it to the database
+		_, err = setupTargetDatabase(config, targetContainerID)
```

Comment:
Does the target need to be shut down?

Reply:
That is done already outside of the function. It's not nice I know ;)